exchanges/poloniex/poloniex_futures.go

```diff
@@ -71,14 +77,14 @@ func (e *Exchange) PlaceFuturesOrder(ctx context.Context, arg *FuturesOrderReque
 	if arg.PositionSide == order.UnknownSide {
 		return nil, order.ErrSideIsInvalid
 	}
-	if arg.OrderType == "" {
+	if arg.OrderType == orderType(order.UnknownType) {
 		return nil, order.ErrTypeIsInvalid
 	}
 	if arg.Size <= 0 {
 		return nil, limits.ErrAmountBelowMin
 	}
 	var resp *FuturesOrderIDResponse
-	if err := e.SendAuthenticatedHTTPRequest(ctx, exchange.RestSpot, request.UnAuth, http.MethodPost, tradePathV3+"order", nil, arg, &resp); err != nil {
+	if err := e.SendAuthenticatedHTTPRequest(ctx, exchange.RestSpot, request.Auth, http.MethodPost, tradePathV3+"order", nil, arg, &resp); err != nil {
```

Comment:
`e.SendAuthenticatedHTTPRequest` should not use `request.Auth` as a param.
Why are you sending that as a rate limit? Why aren't the rate limits defined in ratelimit.go for this?